### PR TITLE
Project Initialized

### PR DESCRIPTION
**Pull request by harshendrashah**
Closes #2

### CODE REVIEW

**Comment by yashLadha** [comment]
@hshah11 There is already a PR for the Initialisation, merging this will lead to conflicts. @Monal5031 Views?

**Comment by Monal5031** [comment]
@yashLadha sir I think this one should be merged, since this PR has more file changes compared to previous ones. We can simply copy-paste the changes you made in #1 .

Please use issues next time :sweat:

**Comment by yashLadha** [comment]
As you like :+1:

**Comment by Monal5031** [comment]
btw @yashLadha sir there is no gitignore here, maybe there are something which we shouldn't push. Idk much about android can you please tell if there are such files which we don't need to push?

File: app/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+/build
```
Comment (Monal5031) [inline app/.gitignore]: this should be `build/` I can see the build files are pushed.

**Review by Monal5031 — CHANGES_REQUESTED** [review]

**Comment by yashLadha** [comment]
@Monal5031 the `.idea/` is not of relevance to the code and its subsequent directories, they should be part of `.gitignore`

**Comment by Monal5031** [comment]
@yashLadha sir please review once.

**Comment by Monal5031** [comment]
ping @yashLadha @neelanshsahai please review and merge.
Get things going ASAP please.

**Pull request closed without merging**